File: CarAgency/ClientApp/src/domain/models/personas/EstadoCivil.ts
import  IEstadoCivil   from "domain/interfaces/personas/IEstadoCivil";

export default class EstadoCivil implements IEstadoCivil {
    EstadoCivilId? : number | null;
    Descripcion!: string;

    public constructor(init?: Partial<EstadoCivil>){
        Object.assign(this, init);
    };

}

File: Domain/Models/Personas/EstadoCivil.cs
﻿using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Threading.Tasks;

namespace Domain.Models.Personas
{
    public class EstadoCivil : ClaseBase
    {
        [Key]
        public int EstadoCivilId { get; set; }

        [Column(TypeName = "nvarchar(50)")]
        public string Descripcion { get; set; }
    }
}
